worlds/animal_well/locations.py

```diff
@@ -183,6 +186,124 @@ class AWLocationData(NamedTuple):
+    lname.fruit_24.value: AWLocationData(1024, ByteSect.fruits, 24, ["Fruits"], AWTracker(130, 0, 0, 0, 16)),
+    lname.fruit_25.value: AWLocationData(1025, ByteSect.fruits, 25, ["Fruits"], AWTracker(174, 0, 0, 0, 6)),
+    lname.fruit_26.value: AWLocationData(1026, ByteSect.fruits, 26, ["Fruits"], AWTracker(174, 0, 0, 0, 7)),
+    lname.fruit_27.value: AWLocationData(1027, ByteSect.fruits, 27, ["Fruits"], AWTracker(130, 0, 0, 0, 17)),
+    lname.fruit_28.value: AWLocationData(1028, ByteSect.fruits, 28, ["Fruits"], AWTracker(130, 0, 0, 0, 18)),
+    lname.fruit_29.value: AWLocationData(1029, ByteSect.fruits, 29, ["Fruits"], AWTracker(130, 0, 0, 0, 19)),
+    lname.fruit_30.value: AWLocationData(1030, ByteSect.fruits, 30, ["Fruits"], AWTracker(130, 0, 0, 0, 20)),
+    lname.fruit_31.value: AWLocationData(1031, ByteSect.fruits, 31, ["Fruits"], AWTracker(174, 0, 0, 0, 8)),
+    lname.fruit_32.value: AWLocationData(1032, ByteSect.fruits, 32, ["Fruits"], AWTracker(174, 0, 0, 0, 9)),
+    lname.fruit_33.value: AWLocationData(1033, ByteSect.fruits, 33, ["Fruits"], AWTracker(130, 0, 0, 0, 21)),
+    lname.fruit_34.value: AWLocationData(1034, ByteSect.fruits, 34, ["Fruits"], AWTracker(130, 0, 0, 0, 22)),
+    lname.fruit_35.value: AWLocationData(1035, ByteSect.fruits, 35, ["Fruits"], AWTracker(174, 0, 0, 0, 10)),
+    lname.fruit_36.value: AWLocationData(1036, ByteSect.fruits, 36, ["Fruits"], AWTracker(130, 0, 0, 0, 23)),
+    lname.fruit_37.value: AWLocationData(1037, ByteSect.fruits, 37, ["Fruits"], AWTracker(130, 0, 0, 0, 24)),
+    lname.fruit_38.value: AWLocationData(1038, ByteSect.fruits, 38, ["Fruits"], AWTracker(130, 0, 0, 0, 25)),
+    lname.fruit_39.value: AWLocationData(1039, ByteSect.fruits, 39, ["Fruits"], AWTracker(174, 0, 0, 0, 11)),
+    lname.fruit_40.value: AWLocationData(1040, ByteSect.fruits, 40, ["Fruits"], AWTracker(174, 0, 0, 0, 12)),
+    lname.fruit_41.value: AWLocationData(1041, ByteSect.fruits, 41, ["Fruits"], AWTracker(174, 0, 0, 0, 13)),
+    lname.fruit_42.value: AWLocationData(1042, ByteSect.fruits, 42, ["Fruits"], AWTracker(130, 0, 0, 0, 26)),
+    lname.fruit_43.value: AWLocationData(1043, ByteSect.fruits, 43, ["Fruits"], AWTracker(174, 0, 0, 0, 14)),
+    lname.fruit_44.value: AWLocationData(1044, ByteSect.fruits, 44, ["Fruits"], AWTracker(130, 0, 0, 0, 27)),
+    lname.fruit_45.value: AWLocationData(1045, ByteSect.fruits, 45, ["Fruits"], AWTracker(130, 0, 0, 0, 28)),
+    lname.fruit_46.value: AWLocationData(1046, ByteSect.fruits, 46, ["Fruits"], AWTracker(130, 0, 0, 0, 29)),
+    lname.fruit_47.value: AWLocationData(1047, ByteSect.fruits, 47, ["Fruits"], AWTracker(130, 0, 0, 0, 30)),
+    lname.fruit_48.value: AWLocationData(1048, ByteSect.fruits, 48, ["Fruits"], AWTracker(130, 0, 0, 0, 31)),
+    lname.fruit_49.value: AWLocationData(1049, ByteSect.fruits, 49, ["Fruits"], AWTracker(174, 0, 0, 0, 15)),
+    lname.fruit_50.value: AWLocationData(1050, ByteSect.fruits, 50, ["Fruits"], AWTracker(174, 0, 0, 0, 16)),
```

Comment:
```suggestion
    lname.fruit_48.value: AWLocationData(1048, ByteSect.fruits, 48, ["Fruits"], AWTracker(130, 0, 0, 0, 15)),
    lname.fruit_49.value: AWLocationData(1049, ByteSect.fruits, 49, ["Fruits"], AWTracker(174, 0, 0, 0, 16)),
    lname.fruit_50.value: AWLocationData(1050, ByteSect.fruits, 50, ["Fruits"], AWTracker(130, 0, 0, 0, 31)),
```
This should fix that bug that 8 mentioned -- in the office and can't test it atm